src/templates/semester_status_banner.html

```
@@ -17,12 +20,16 @@
         </div>
-    {% elif config.active_semester.has_ended %}
-        <div class="alert alert-warning" id="semester-status-banner">
-            <i class="fa fa-fw fa-calendar-o"></i>
-            <strong>Semester {{ config.active_semester }} ended on {{ config.active_semester.last_day }}.</strong>
-            <a href="{% url 'courses:semester_archive' config.open_semester_id %}" class="alert-link">Archive it</a>
-            to record final marks and free up your students' seats.
-        </div>
+    {% else %}
+        {% for semester in config.open_semesters %}
+            {% if semester.has_ended %}
+            <div class="alert alert-warning" id="semester-status-banner">
```

HTMLHint (1.9.2)

[error] The id value [ semester-status-banner ] must be unique.

(id-unique)